purchase_order_general_discount/models/purchase_order.py

```diff
@@ -64,5 +64,5 @@ def _get_view(self, view_id=None, view_type="form", **options):
                     f"{{'default_{discount_field}': general_discount, ",
                     1,
                 )
-                order_line_field.attrib["context"] = context
+                order_line_field.set("context", context)
```

Comment:
Update the context from the view, using `operation="update"`. For example:
![image](https://github.com/user-attachments/assets/7657825a-01ad-4dd3-a243-4dcf7f39de5a)